new/LIT6298MGMostVenerated.xml

```diff
@@ -0,0 +1,106 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6298MGMostVenerated" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: A virtuous monk witnesses
+                    that St. George is the most venerated of all saints</title>
```

Comment:
I think we are going for St. without a full stop (i.e. St) but this probably needs to be consistent in the catalogue, not sure if it matters for these records!